### PR TITLE
Feature/mds 2524 compliance data on minespace

### PR DESCRIPTION
**Pull request by JustinMacaulay**
Displays Compliance data on minespace. 
All columns are sortable and allows the mines to hide or display the closed orders.

### CODE REVIEW

File: services/core-api/app/api/services/NRIS_API_service.py [hunk 1]
```diff
@@ -61,7 +62,7 @@ def _process_NRIS_data(raw_data):
         'last_inspector': None,
         'num_open_orders': 0,
         'num_overdue_orders': 0,
-        #'section_35_orders': 0, no aggregate, FE filters will show where violation = 35
+                                                                          #'section_35_orders': 0, no aggregate, FE filters will show where violation = 35
```
Comment (mmumby) [inline services/core-api/app/api/services/NRIS_API_service.py]: odd formatting

File: services/core-web/common/constants/strings.js [hunk 2]
```diff
@@ -76,3 +76,14 @@ export const INCIDENT_FOLLOWUP_ACTIONS = {
   none: "NO",
   unknown: "HUK",
 };
+
+export const RETURN_PAGE_TYPE = {
+  LOGIN: "login",
+  SITEMINDER_LOGOUT: "smlogout",
+  LOGOUT: "logout",
+};
+
```
Comment (mmumby) [inline services/core-web/common/constants/strings.js]: Core shouldn't need this const, this is for the different auth in MineSpace

File: services/minespace-web/src/components/dashboard/mine/inspections/Inspections.js [hunk 3]
```diff
@@ -4,24 +4,47 @@
...
 const propTypes = {
   mine: CustomPropTypes.mine.isRequired,
+  mineComplianceInfo: CustomPropTypes.mineComplianceInfo.isRequired,
+  fetchMineComplianceInfo: PropTypes.func.isRequired,
 };
 
 const defaultProps = {};
 
 export class Inspections extends Component {
   // TODO: Accurately set isLoaded when file is more properly implemented.
```
Comment (mmumby) [inline services/minespace-web/src/components/dashboard/mine/inspections/Inspections.js]: This comment can be removed now

File: services/minespace-web/src/components/dashboard/mine/inspections/InspectionsTable.js [hunk 4]
```diff
@@ -1,39 +1,124 @@
...
+        )}
+      </div>
+    ),
+  },
+  {
+    title: "Order No.",
+    dataIndex: "order_no",
+    key: "order_no",
+    render: (text, record) => <div title="Order #">{record.order_no || "-"}</div>,
+    sorter: (a, b) => (a.order_no > b.order_no ? -1 : 1),
```
Comment (mmumby) [inline services/minespace-web/src/components/dashboard/mine/inspections/InspectionsTable.js]: There is a const for '-', I believe its 'EMPTY_FIELD'

File: services/minespace-web/src/components/dashboard/mine/inspections/InspectionsTable.js [hunk 5]
```diff
@@ -1,39 +1,124 @@
...
-    dataSource={data}
-    locale={{ emptyText: "This mine has no inspection data." }}
-  />
-);
+const transformRowData = (orders) =>
+  orders.map((order) => ({
+    key: order.order_no,
+    ...order,
+  }));
+
```
Comment (mmumby) [inline services/minespace-web/src/components/dashboard/mine/inspections/InspectionsTable.js]: It doesn't look like 'record.key' is being referenced anywhere, You can get rid of the 'transformTowData' function altogether and pass ordered directly into the table
`dataSource={filterClosedOrders(props.orders, showClosedOrders)}`

File: services/core-api/app/api/services/NRIS_API_service.py (hunk 1 above)
Comment (JustinMacaulay) [reply services/core-api/app/api/services/NRIS_API_service.py]: This is the the yapf formatting we can change the config but that is the way it is configured to format comments.

File: services/core-web/common/constants/strings.js (hunk 2 above)
Comment (JustinMacaulay) [reply services/core-web/common/constants/strings.js]: put this back in the minespace specific strings.

File: services/minespace-web/src/components/dashboard/mine/inspections/Inspections.js (hunk 3 above)
Comment (JustinMacaulay) [reply services/minespace-web/src/components/dashboard/mine/inspections/Inspections.js]: removed.

File: services/minespace-web/src/components/dashboard/mine/inspections/InspectionsTable.js (hunk 4 above)
Comment (JustinMacaulay) [reply services/minespace-web/src/components/dashboard/mine/inspections/InspectionsTable.js]: We do not, EMPTY_FIELD is "N/A"

File: services/minespace-web/src/components/dashboard/mine/inspections/InspectionsTable.js (hunk 5 above)
Comment (JustinMacaulay) [reply services/minespace-web/src/components/dashboard/mine/inspections/InspectionsTable.js]: removed.

**Pull request merged**